massgov/openmass
# dp 26357 fix js error

<img width="1723" alt="Screen Shot 2022-11-08 at 3 31 32 PM" src="https://user-images.githubusercontent.com/5789411/200903590-f3adf867-5690-4bbb-b169-831b5fa8e182.png">

On Mass.gov error pages, the Mayflower JS is throwing errors due to elements returning null. See fixes in Mayflower --> https://github.com/massgov/mayflower/pull/1696



To Test:
1. Go to the links below
>Before:https://www.mass.gov/testtesttest
>After: https://pr1782-hergim6shr6mnlcriekwcwalbkkya4th.tugboatqa.com/jigd

3. Open the browser console, and make sure the errors are resolved 
4. Tab around on the page, and make sure no errors are thrown
